### ISSUE TITLE
Unable to remove menu entries

### ISSUE THREAD

**Issue by TriMoon** [ValveSoftware/steam-for-linux#6390]
#### Your system information

* Steam client version: **Jun 17 2019, 23:31:27 (Pkg,ver.:1560817063)**
* Distribution: **Kubuntu 19.04**
* Opted into Steam client beta?: Was YES, but now changed to NO (CS-GO wont start anymore)
* Have you checked for system updates?: Yes

#### Please describe your issue in as much detail as possible:
I am unable to remove games i have tried in the past from the menu you get when right-clicking the icon in the taskbar near the clock.
I have tried removing all `~/Steam` and `~/.steam` folders from my user account, that made steam reinitialize and make me re-download the games i want to play.
But the menu entries stay for the games i DONT want anymore.

Please provide an option to clean this menu listing from within the client.
Until then i would like to know how to purge that listing myself.

#### Steps for reproducing this issue:
See above

Was originally posted at: https://help.steampowered.com/en/wizard/HelpRequest/HT-F9WY-3RR8-MDYX

**Comment by dearlygparted** [comment]
Then menu only holds the most recent 5 games. A workaround would be to play 5 other games and bump the games you don't want off the list.

**Comment by TriMoon** [comment]
> The menu only holds the 5 most recent games. A workaround would be to play 5 other games and bump the games you don't want off the list.

:rofl: well thanks but the case is that i only play 3 frequently... :wink:

**Comment by Protonator** [comment]
Yep, there are not only previously played or installed games in the list, but also recently purchased titles or things like Proton. Would be great to have an option to customize the list or remove entries.
